ex/high_throughput_inference/mli_driver.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
Don't forget to add your licenses to your new files

Reply:
Done